src/sumo_store_riak.erl

```diff
@@ -373,13 +373,13 @@ fetch_docs(DocName, Conn, Bucket, Keys, Opts) ->
   end, [], Keys).
 
 -spec delete_map(
-  connection(), bucket(), key(), options()
+  connection(), bucket_and_type(), key(), options()
 ) -> ok | {error, term()}.
 delete_map(Conn, Bucket, Key, Opts) ->
   riakc_pb_socket:delete(Conn, Bucket, Key, Opts).
 
 -spec update_map(
-  connection(), bucket(), key() | undefined, riakc_map:crdt_map(), options()
+  connection(), bucket_and_type(), key() | undefined, riakc_map:crdt_map(), options()
```

Comment:
Line 382 is too long:   connection(), bucket_and_type(), key() | undefined, riakc_map:crdt_map(), options().